scripts/run-test-suite.sh

```diff
@@ -47,7 +47,7 @@ test_image()
     echo "========================================="
     echo "Testing $LISP..."
 
-    mpiexec -n 4 $IMAGE
+    mpiexec -n 4 --oversubscribe $IMAGE
```

Comment:
I am but a humble peasant with only two physical CPU cores to my name. Without the `--oversubscribe` option, `mpiexec` complains thusly:

```
There are not enough slots available in the system to satisfy the 4
slots that were requested by the application:

  ./scripts/sbcl.image

Either request fewer slots for your application, or make more slots
available for use.
```

Reply:
It turns out that `--oversubscribe` is apparently a non-standard `mpiexec` option, or at least the `mpiexec` that comes with MPICH doesn't support it. I guess will drop this in the next patch set, since most people in 2019 probably have at least 4 cores :-|.